src/content/en/fundamentals/engage-and-retain/push-notifications/permissions-subscriptions.markdown

```diff
@@ -0,0 +1,174 @@
+---
+layout: shared/narrow
+title: "Requesting permission and subscribing users"
+description: "Requesting permission for and subscribing users to notifications requires as 
+light a touch as showing them notifications."
+published_on: 2016-07-01
+updated_on: 2016-07-01
+order: 20
+translation_priority: 0
+authors:
+  - josephmedley
+notes:
+  clarity: "For the sake of clarity, this example excludes a number of feature checks that you should always perform. You can view the original code in it's entirety in our <a href='https://github.com/GoogleChrome/samples/tree/gh-pages/push-messaging-and-notifications'>GitHub samples repo</a>."
+---
+
+<p class="intro">
+Requesting permission for and subscribing users to notifications requires as
+light a touch as showing them notifications.
+</p>
+
+{% include shared/toc.liquid %}
+
+## Check permissions {#check-permissions}
+
+Always check for an existing permission when the page loads. If the permission
```

Comment:
Re-reading this, this paragraph reads a little weird to me, and do we want devs to check permissions on page load? I thought we frowned upon that.